tools/clang/lib/SPIRV/DebugTypeVisitor.cpp

```diff
@@ -269,6 +269,14 @@ bool DebugTypeVisitor::visitInstruction(SpirvInstruction *instr) {
       if (spirvType) {
         SpirvDebugInstruction *debugType = lowerToDebugType(spirvType);
         debugInstr->setDebugType(debugType);
+        if (auto *compositeInfo =
```

Comment:
is this whole part with the loop and the setDefaultDebugInfo really needed? Looks like line 46-56 is doing the same thing.

Reply:
Nice catch. I think I was insane.